### ISSUE TITLE
Syntax highlighting does not work for certain file types (eg. .json, .ts)

### ISSUE THREAD

**Issue by rewko** [jakubgarfield/Bonobo-Git-Server#450]
FileDisplayHandler renders brush for only smaller set of file extensions - otherwise it sets explicitly "no-highlight".
Highlight.js used on on the client is able to [auto detect](https://highlightjs.org/usage/) correct syntax, but this feature is turned of because "no-higlight" flag rendered in HTML.

At least the set of languages supported by included highlight.js should be the same as those checked on server. But if auto detection works fine on client why bother with that on the server?

**Comment by willdean** [comment]
I looked into this - and for the examples you gave, returning null rather than NoHighlight definitely works OK.  

However, the code which processes all this stuff in Repository.BrowseBlob feels extremely fragile to me - it makes several decisions which it then overrules in various ways, and I am reluctant to risk upsetting it until there's some test coverage.  

So, I've just added explicit .json and .ts support, which I know isn't quite what was wanted but should be a bit helpful.
